### ISSUE TITLE
NullPointerException in MessageBirdServiceImpl

### ISSUE THREAD

**Issue by rmacqueen** [messagebird/java-rest-api#114]
I am encountering a null pointer exception when calling `startConversation` on the MessageBirdClient

```
java.lang.NullPointerException: null
	at com.messagebird.MessageBirdServiceImpl.getErrorReportOrNull(MessageBirdServiceImpl.java:545) ~[messagebird-api-3.0.13.jar!/:na]
	at com.messagebird.MessageBirdServiceImpl.handleHttpFailStatuses(MessageBirdServiceImpl.java:245) ~[messagebird-api-3.0.13.jar!/:na]
	at com.messagebird.MessageBirdServiceImpl.getJsonData(MessageBirdServiceImpl.java:236) ~[messagebird-api-3.0.13.jar!/:na]
	at com.messagebird.MessageBirdServiceImpl.sendPayLoad(MessageBirdServiceImpl.java:176) ~[messagebird-api-3.0.13.jar!/:na]
	at com.messagebird.MessageBirdServiceImpl.sendPayLoad(MessageBirdServiceImpl.java:166) ~[messagebird-api-3.0.13.jar!/:na]
	at com.messagebird.MessageBirdClient.startConversation(MessageBirdClient.java:951) ~[messagebird-api-3.0.13.jar!/:na]
```

It looks like the null pointer is being referenced at this line https://github.com/messagebird/java-rest-api/blob/master/api/src/main/java/com/messagebird/MessageBirdServiceImpl.java#L545

**Comment by denizkilic** [comment]
Hi @rmacqueen,
Thanks for reporting this issue. @olimpias solved it. A new version will be released soon.

**Comment by rmacqueen** [comment]
Great, thanks

**Comment by denizkilic** [comment]
Hi @rmacqueen,
The issue was fixed in Release 3.0.14. We are ready to welcoming more issues if you have.
